bodhi/tests/server/services/test_updates.py

```diff
@@ -3583,7 +3582,7 @@ def test_disabled_button_for_autopush(self, *args):
         resp = self.app.get(f"/updates/{resp.json['alias']}", headers={'Accept': 'text/html'})
         assert 'text/html' in resp.headers['Content-Type']
         assert nvr in resp
-        assert 'Stable by Karma' not in resp
+        assert 'disabled' in resp
```

Comment:
Too generic.
Since `Stable by Karma` is now always displayed, we could use the tooltip text:
`assert 'The update will not be automatically pushed to stable by karma' in resp`